code/modules/telesci/gps.dm

```diff
@@ -50,19 +52,34 @@ var/list/GPS_list = list()
 	popup.set_title_image(user.browse_rsc_icon(src.icon, src.icon_state))
 	popup.open()
 
+/obj/item/device/gps/examine(mob/user)
+	if (Adjacent(user) || isobserver(user))
+		src.attack_self(user)
+	else
+		..()
```

Comment:
The idea is that it doesn't print the description of the GPS if the window opens, paper works in the same way currently